src/test/java/net/snowflake/client/config/SFClientConfigParserTest.java

```diff
@@ -79,7 +79,8 @@ public void testloadSFClientConfigWithEnvVar() {
 
   @Test
   public void testloadSFClientConfigWithDriverLoaction() {
-    String configLocation = getConfigFilePathFromJDBCJarLocation();
+    String configLocation =
```

Comment:
Does this test cover the Windows path too?

Reply:
When we run this test on windows, it should cover the windows logic.

Reply:
added a test for windows by mocking system property!

Reply:
Can you confirm on this please?

Reply:
I added a new test `testgetConfigFileNameFromJDBCJarLocationForWindows` to verify the Windows path. That should be enough right?